test/system/pledge_fulfillment_test.rb

```diff
@@ -15,7 +15,6 @@ class PledgeFulfillmentTest < ApplicationSystemTestCase
     @nonpledged_pt = create :patient, clinic: @clinic,
                                       appointment_date: 2.weeks.from_now,
                                       fund_pledge: 500
-    @fulfillment = create :fulfillment, patient: @pledged_pt
   end
```

Comment:
Guessing that these got removed because we're creating fulfillment with an object callback? 👍 for cleaning up my bad test code, thank you!

Reply:
That seems a likely explanation! I just noticed it was unneeded on the console and gave deleting it in the test a shot. Still functioned!